src/busesApp/router.js

```diff
@@ -52,4 +53,11 @@ busesRouter.delete(
 	asyncHandler(deleteBusHandler)
 );
 
+busesRouter.get(
+	'/:plate/driver',
```

Comment:
about that I could really use some ideas I found some resources about resources association in REST API that you can check out [here](https://stackoverflow.com/questions/11337823/restful-api-design-with-associations) and [here](https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design) where they used requirements inside route path but I get how it can be confusing 
I will look into it